src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -25,11 +25,14 @@ class PrisonerIepLevelReviewService(
   private val iepLevelRepository: IepLevelRepository,
   private val authenticationFacade: AuthenticationFacade
 ) {
-  suspend fun getPrisonerIepLevelHistory(bookingId: Long, useNomisData: Boolean = true): IepSummary {
+  suspend fun getPrisonerIepLevelHistory(bookingId: Long, useNomisData: Boolean = true, withDetails: Boolean = true): IepSummary {
     return if (useNomisData) {
-      prisonApiService.getIEPSummaryPerPrisoner(listOf(bookingId)).first()
+      val result = prisonApiService.getIEPSummaryPerPrisoner(listOf(bookingId)).first()
+      result.copy(
+        iepDetails = if (withDetails) result.iepDetails else emptyList()
+      )
     } else {
-      buildIepSummary(prisonerIepLevelRepository.findAllByBookingIdOrderBySequenceDesc(bookingId))
+      buildIepSummary(prisonerIepLevelRepository.findAllByBookingIdOrderBySequenceDesc(bookingId), withDetails)
```

Comment:
Contemplated using `PrisonerIepLevelRepository.findOneByBookingIdAndCurrentIsTrue` but we need to calculate `daysSinceReview` which requires the list of bookings